packaging/Makefile

```diff
@@ -26,21 +26,20 @@ docker-push:
 	docker push $(IMAGE)
 
 plugins:
-	CUR=$(shell pwd) ; GOPATH=$(shell pwd)/build ;\
-		go get -d github.com/skipper-plugins/opentracing ;\
-		cd build/src/github.com/skipper-plugins/opentracing ;\
-		glide install -v;\
+	mkdir -p build/; cd build ;\
+	git clone --branch=go-modules https://github.com/mikkeloscar/opentracing.git ;\
```

Comment:
This should be changed once https://github.com/skipper-plugins/opentracing/pull/10 is merged.

Reply:
please change it

Reply:
Done